google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/Date.java

```diff
@@ -68,7 +70,39 @@ public static Date parseDate(String date) {
     int dayOfMonth = Integer.parseInt(matcher.group(3));
     return new Date(year, month, dayOfMonth);
   }
+  
+  /**
+  * @param date
+  * @return LocalDate
+  */
+  public static LocalDate toDate(Date date){
+	return LocalDate.of(date.year, date.month, date.dayOfMonth);  
+  }
+   
+  /**
+  * @param date
+  * @return Date
+  */
+  public static Date toDate(LocalDate date){
+    return new Date(date.getYear(), date.getMonthValue(), date.getDayOfMonth());
+  }
 
+  /**
+  * @param date
+  * @return LocalDate
+  */
+  public static LocalDate fromDate(Date date){
```

Comment:
`Date::fromDate` feels odd when it is constructing a `LocalDate`. The name should be different.